scripts/server.sh

```diff
@@ -19,6 +19,6 @@ then
     then
         usage
     else
-      npm run start
+      vagrant ssh -c "cd /vagrant/dist/ && python -m SimpleHTTPServer 8000"
```

Comment:
Is there a reason we need to execute this outside of the VM? The convention we have been following for these scripts (`update`, `server`, etc.) has been to execute them within the context of the VM, so we shouldn't need to wrap these commands with `vagrant ssh ...`, right?